### PR TITLE
Mission statement complete

### PR DESCRIPTION
**Pull request by gregkorte**

### CODE REVIEW

**Comment by Saul3d** [comment]
We are missing and index.html file.
Which will need a link to the employees.html

File: employees.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```
Comment (Saul3d) [inline employees.html]: This file should be in a HTML directory

Comment (gregkorte) [reply employees.html]: Fixed all issues.

**Comment by gregkorte** [comment]
Fixed all PR issues.

**Comment by Saul3d** [comment]
👍

**Pull request merged**